content/static/css/main.css

```diff
@@ -0,0 +1 @@
ign:center}.text-4xl{font-size:2.25rem;line-height:2.5rem}.text-2xl{font-size:1.5rem;line-height:2rem}.text-base{font-size:1rem;line-height:1.5rem}.text-xs{font-size:.75rem;line-height:1rem}.text-sm{font-size:.875rem;line-height:1.25rem}.text-lg{font-size:1.125rem}.text-lg,.text-xl{line-height:1.75rem}.text-xl{font-size:1.25rem}.text-3xl{font-size:1.875rem;line-height:2.25rem}.text-6xl{font-size:3.75rem;line-height:1}.font-light{font-weight:300}.font-thin{font-weight:100}.font-extrabold{font-weight:800}.font-bold{font-weight:700}.uppercase{text-transform:uppercase}.italic{font-style:italic}.text-white{--tw-text-opacity:1;color:rgb(255 255 255/var(--tw-text-opacity))}.text-lime-600{--tw-text-opacity:1;color:rgb(101 163 13/var(--tw-text-opacity))}.text-black{--tw-text-opacity:1;color:rgb(0 0 0/var(--tw-text-opacity))}.underline{-webkit-text-decoration-line:underline;text-decoration-line:underline}.shadow-xl{--tw-shadow:0 20px 25px -5px #0000001a,0 8px 10px -6px #0000001a;--tw-shadow-colored:0 20px 25px -5px var(--tw-shadow-color),0 8px 10px -6px var(--tw-shadow-color);box-shadow:var(--tw-ring-offset-shadow,0 0 #0000),var(--tw-ring-shadow,0 0 #0000),var(--tw-shadow)}.invert{--tw-invert:invert(100%)}.filter,.invert{filter:var(--tw-blur) var(--tw-brightness) var(--tw-contrast) var(--tw-grayscale) var(--tw-hue-rotate) var(--tw-invert) var(--tw-saturate) var(--tw-sepia) var(--tw-drop-shadow)}body{font-family:Roboto}.drop-color{background:#fff;background:linear-gradient(0deg,#fff,#fff0)}@media (min-width:640px){.sm\:mx-auto{margin-left:auto;margin-right:auto}.sm\:h-28{height:7rem}.sm\:h-80{height:20rem}.sm\:h-5{height:1.25rem}.sm\:w-2\/3{width:66.666667%}.sm\:w-1\/3{width:33.333333%}.sm\:px-10{padding-left:2.5rem;padding-right:2.5rem}.sm\:px-5{padding-left:1.25rem;padding-right:1.25rem}.sm\:px-2{padding-left:.5rem;padding-right:.5rem}.sm\:pr-10{padding-right:2.5rem}.sm\:pt-0{padding-top:0}.sm\:text-sm{font-size:.875rem;line-height:1.25rem}}@media (min-width:768px){.md\:order-last{order:9999}.md\:mx-auto{margin-right:auto}.md\:ml-auto,.md\:mx-auto{margin-left:auto}.md\:block{display:block}.md\:h-72{height:18rem}.md\:h-36{height:9rem}.md\:h-20{height:5rem}.md\:w-1\/2{width:50%}.md\:w-1\/3{width:33.333333%}.md\:w-2\/3{width:66.666667%}.md\:w-1\/5{width:20%}.md\:w-auto{width:auto}.md\:w-6{width:1.5rem}.md\:px-2{padding-left:.5rem;padding-right:.5rem}.md\:px-1{padding-left:.25rem;padding-right:.25rem}.md\:pr-2{padding-right:.5rem}.md\:text-xs{font-size:.75rem;line-height:1rem}.md\:text-4xl{font-size:2.25rem;line-height:2.5rem}.md\:text-xl{font-size:1.25rem;line-height:1.75rem}.md\:text-base{font-size:1rem;line-height:1.5rem}}@media (min-width:1024px){.lg\:flex{display:flex}.lg\:h-60{height:15rem}.lg\:w-1\/3{width:33.333333%}.lg\:w-1\/2{width:50%}.lg\:w-1\/5{width:20%}.lg\:text-lg{font-size:1.125rem;line-height:1.75rem}}@media (min-width:1280px){.xl\:w-1\/6{width:16.666667%}.xl\:w-4\/6{width:66.666667%}.xl\:p-5{padding:1.25rem}.xl\:pr-5{padding-right:1.25rem}}
```

Comment:
Are we sure we want to commit/track the minified version?
Maybe what we want instead is to install it as a npm dependency and then link to it from the `node_modules/` folder?

Reply:
Nope, it wasn't intentional 😆, addressed it.

Instead, I don't think that we can link it from the `node_modules` folder, as this one is built every time an `*.html` file into the `templates` folder changes (Configuration is located at `tailwind.config.js`)

Basically, is not the full minified `tailwindcss`, but only contains the classes used on our pages, in order to keep the file size minimal.

Reply:
Oh right, that makes sense indeed :+1: